pkg/kmetrics/fieldcount.go

```diff
@@ -268,9 +268,6 @@ func k8sMetadataCount(k *types.Kustomization) map[string]int {
 	if k.Namespace != "" {
 		result["Namespace"] = 1
 	}
-	if len(k.CommonLabels) > 0 {
```

Comment:
Why was this removed?

Reply:
I guess I don't really know what this field is used for or if it's safe to remove it. Is this because we updated Kustomize at the same time as Go? Is it necessary to update both at the same time?

Reply:
> Deprecated: Use the Labels field instead, which provides a superset of the functionality of CommonLabels. CommonLabels to add to all objects and selectors.

If we're parsing the user's kustomize file and using it to render then we probably need to keep passing both to kustomize until it's removed. 

So add a linter comment to ignore it, and maybe log an error that it's deprecated so the user is made aware.

Reply:
Agreed, done